cloudsql/mysql/database-sql/README.md

```diff
@@ -64,10 +66,16 @@ Start-Process -filepath "C:\<path to proxy exe>" -ArgumentList "-instances=<proj
 ### Launch proxy with Unix Domain Socket
 NOTE: this option is currently only supported on Linux and Mac OS. Windows users should use the [Launch proxy with TCP](#launch-proxy-with-tcp) option.
 
-To use a Unix socket, you'll need to create the `/cloudsql` directory and give write access to the user running the proxy. Use these commands to create the directory and set permissions:
+To use a Unix socket, you'll need to create a directory and give write access to the user running the proxy. For example:
 ```bash
-sudo mkdir /cloudsql
-sudo chown -R $USER /cloudsql
+sudo mkdir /path/to/the/new/directory
```

Comment:
nit: Seems like we could just say `./cloudsql` and not push the developer to making another decision on this.